frontend/src/util/index.tsx

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+import React from 'react';
+
+const format_date = (
+  date?: string | Date,
+  default_format?: string,
```

Comment:
is default_format ever used in the codebase? if not we dont need to account for it